### DISCUSSION TITLE
Only thing thats scraped is Avatar & Header image  and user_data.db

### DISCUSSION THREAD

**Discussion by dargpat** [DIGITALCRIMINALS/OnlyFans#644] [Q&A]
Script seems to run fine except its not downloading any other files.  Can someone please help.  I haven't made any changes to the config file.  Am I missing something here?  Cant seem to get this to download anything other than whats listed in the subject.

**Comment by MajesticMagenta** [comment]
I had the same problem but then I checked my sess and x b_c and it was different I changed it and it worked ^_^

**Reply by dargpat** [reply]
I don't know what happened, but it started working.  I did keep updating my auth.json file over and over and next thing I know was it started downloading. 🤷

**Comment by Soygen** [comment]


**Reply by dargpat** [reply]
Apparently, the script scrapes those listed in the subject first without downloading what you really want.  It even scrapes the accounts you received messages from in your inbox, which could be possibly hundreds of accounts.  I disabled the message scraping in the configuration file then tried it again.  Since it was fewer accounts to scrape from the downloading started after aboout 10 mins (depending on size of content each creator has). Let me know if this helped.

**Reply by dargpat** [reply]
Glad to help!

**Reply by jamesmiller87366** [reply]
how did you disable it?

**Reply by OoAlvaRoO** [reply]
How can I disable message scraping? Sorry I am new in this kind of things @dargpat

**Comment by LouisChoiseul** [comment]
Hi, I was with the same problem, only downloads the avatar, header image and user_data. I disabled the message scraping as insctructed and started downloading the content, but it always stops after the first one and get stuck at 5-10%. I retried multiple times and it always get stuck after the first file.
__________________________________________________

PS D:\Download\OF Downloads\OnlyFans> python start_ofd.py
Auth (V1) Attempt 1/10
Welcome [myUsername]
Choose Subscriptions: 0 = All | 1 = subscription1 | 2 = subscription2 | 3 = subscription3 | 4 = subscription4 | 5 = ...
2
Scraping Subscriptions
Scrape Processing
Name: subscription2
Choose Medias: 0 = All | 1 = Images | 2 = Videos | 3 = Audios | 4 = Texts
0
Type: Stories
Type: Posts
Scrape Attempt: 1/100
Type: Archived Posts
Processing Scraped Posts
100%|████████████████████████████████████████████████████████████████████████████████| 19/19 [00:00<00:00, 1218.19it/s]
Processing metadata.
Finished processing metadata.
Renaming files.
100%|████████████████████████████████████████████████████████████████████████████████| 19/19 [00:00<00:00, 1217.58it/s]
Type: Archived
Type: Chats
Type: Messages
Type: Highlights
Type: MassMessages
Scrape Completed

Scraping Paid Content
Scraping - subscription1 | 1 / 18
Scraping - subscription2 | 2 / 18
Processing Scraped Posts
100%|████████████████████████████████████████████████████████████████████████████████| 19/19 [00:00<00:00, 1216.61it/s]
Processing metadata.
Finished processing metadata.
Renaming files.
100%|████████████████████████████████████████████████████████████████████████████████| 19/19 [00:00<00:00, 1216.06it/s]
Scraping - subscription3 | 3 / 18
Scraping - subscription4 | 4 / 18
Scraping - ...

Download Processing
Name: subscription2 | Type: Posts | Count: 21 | Directory: .sites

 12%|█████████                                                                    | 2.49M/21.2M [00:14<00:01, 9.79MB/s]

**Reply by dargpat** [reply]
Is it stopping on the same user(subscription) every time?  If so, I would try blocking that account to see if it does the same thing on the next account?  I've ran into a simular problem before using a different script.  And after I blocked the account it kept stopping on, everything went fine and continued to scrape the rest.

**Reply by bumblebeeman85** [reply]
It doesn’t matter what user I choose. Happens with everyone. Free or paid(with active subscription)

**Reply by dargpat** [reply]
Oh ok... might want to wait on a dev to check into that for you.

**Reply by LouisChoiseul** [reply]
Yep, same as @bumblebeeman85, either if I choose a specific subscription or all, it get stuck after the first post.

**Comment by alphabetaman** [comment]
I updated x b_c and disabled messages, and now my downloads get stuck on 0%.. Still trying to get this working for the first time, only got the headers/ avatars to save.
